docs/examples/blind-auction.rst

```diff
@@ -207,12 +218,27 @@ invalid bids.
 
         event AuctionEnded(address winner, uint highestBid);
 
-        /// Modifiers are a convenient way to validate inputs to
-        /// functions. `onlyBefore` is applied to `bid` below:
-        /// The new function body is the modifier's body where
-        /// `_` is replaced by the old function body.
-        modifier onlyBefore(uint _time) { require(block.timestamp < _time); _; }
-        modifier onlyAfter(uint _time) { require(block.timestamp > _time); _; }
+        // Errors that describe failures.
+
+        /// The function has been called too early.
+        /// Try again at `_time`.
+        error TooEarly(uint _time);
+        /// The function has been called too late.
+        /// It cannot be called after `_time`.
+        error TooLate(uint _time);
```

Comment:
Other errors you defined in other places do not have the `_` prefix:
```suggestion
        error TooEarly(uint time);
        /// The function has been called too late.
        /// It cannot be called after `_time`.
        error TooLate(uint time);
```

This reminds me: you should update the Solidity coding style page to mention errors and the recommended naming convention (including parameter names).